tool/site/client-react/src/components/prescriptionActions.tsx

```diff
@@ -7,6 +7,7 @@ interface PrescriptionActionsProps {
   prescriptionId: string
   cancel?: boolean
   release?: boolean
+  returnRelease?: boolean
```

Comment:
is `returnRelease` potentially a confusing name? whats wrong with just `return`?

Reply:
return is a reserved word (as in return from function) so won't compile. Yeah definitely worth considering, agree not in this pr though